bedrock/newsletter/forms.py

```diff
@@ -263,7 +263,10 @@ def __init__(self, newsletters, locale, data=None, *args, **kwargs):
             lang, country = lang.split('-', 1)
         else:
             country = ''
-            regions.insert(0, ('', _lazy('Select country')))
+            if locale.startswith('en'):
```

Comment:
Hmm... perhaps I was wrong... it looks like this won't ever be shown to any of our `en` locales since they all include the country, which pre-selects a country and doesn't show this text. So maybe we should just include a call to the new string here so it can be extracted and translated and then switch to it later?

Reply:
oh, it seemed to work locally... but yeah, I think I follow your logic. The string is already out to localizers so we can just remove this and we'll update later once we've got the l10n coverage.

Reply:
Okay, after looking into it I think we can use the same tag from the template in here:

```python
if lang_file_has_tag('newsletter/existing', locale, 'country_region_122019'):
```

Assuming `newsletter/existing` is the lang file that would get this tag.

Reply:
Oh, and you'd have to include `lang_file_has_tag` in the import from `lib.l10n_utils.dotlang`.